### PR TITLE
auto case OCP-36287

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@huiran0826 @anuragthehatter @weliang1 @rbbratta  PTAL, thanks
logs: 
http://pastebin.test.redhat.com/916187

### CODE REVIEW

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -412,3 +412,18 @@ Feature: SDN related networking scenarios
...
       | Not applying unsafe change: invalid configuration |
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-36287
+  @admin
+  @destructive
+  Scenario: Netnamespace should be recreated after deleting it before the project is deleted
+    Given the env is using "OpenShiftSDN" networkType
+    Given I have a project
+    And admin checks that the "<%= project.name %>" net_namespace exists
```
Comment (liangxia) [inline features/networking/sdn.feature:423]: Is admin here (and below) a must ? or we are OK to use `And I check that ...` ?

Comment (zhaozhanqi) [reply features/networking/sdn.feature:423]: yes, admin is a must since the netnamespace is cluster scope.

**Pull request merged**